package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "clarity-all",
-    "version": "0.10.6",
```

Comment:
Why It's versioned from `0.10.6`?

Reply:
Its not. Thats the deleted line :-)

Reply:
I know but why it's not 0.10.7? Was your branch not rebased?

Reply:
Oh I checked with @jeeyun. It was actually an error in the webpack PR. The package.json file was rebased incorrectly.